### PR TITLE
[WIP]: This pull request adds data from the TESS exoplanet mission

### PR DESCRIPTION
**Pull request by arfon**
Note, if possible, please don't merge this until tomorrow (6th December) which is the go-live date with the new data release.

Some links:
- https://archive.stsci.edu/tess/
- https://heasarc.gsfc.nasa.gov/docs/tess/

### CODE REVIEW

File: datasets/tess.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Resources:
+  - Description: Project data files in a 'Requester Pays' bucket
+    ARN: arn:aws:s3:::stpubdata
+    Region: us-east-1
+    Type: S3 Bucket
+  - Description: Notifications for new data
+    ARN: arn:aws:sns:us-east-1:879230861493:stpubdata
+    Region: us-east-1
+    Type: SNS Topic
+DataAtWork:
```
Comment (arfon) [inline datasets/tess.yaml]: Would it be useful to add these links here too? 

- https://archive.stsci.edu/tess/
- https://heasarc.gsfc.nasa.gov/docs/tess/

File: datasets/tess.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Monthly
+Tags:
+  - astronomy
+  - aws-pds
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files in a 'Requester Pays' bucket
+    ARN: arn:aws:s3:::stpubdata
```
Comment (arfon) [inline datasets/tess.yaml]: These data are in `stpubdata/tess` is it acceptable to have two entries in this registry with the same bucket (the Hubble data are in `stpubdata/hst`)?

Comment (jflasher) [reply datasets/tess.yaml]: Yep, I think that's fine. I would just list the ARN as `arn:aws:s3:::stpubdata/tess` and update to `arn:aws:s3:::stpubdata/hst` in the other file.

File: datasets/tess.yaml (hunk 1 above)
Comment (jflasher) [reply datasets/tess.yaml]: If the links are specifically talking about how to use the data on AWS, then I think they're ok to include in `DataAtWork`. I don't think that's the case though, so I would recommend calling them out in the `Description` section. You can use markdown to make the links look nice.

File: datasets/tess.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  The Transiting Exoplanet Survey Satellite (TESS) is a two-year survey that will discover exoplanets in orbit around bright stars.
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Monthly
+Tags:
+  - astronomy
+  - aws-pds
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files in a 'Requester Pays' bucket
```
Comment (jflasher) [inline datasets/tess.yaml]: Can you add a link to the RP docs here? The field accepts markdown so you can just do [Requester Pays](https://docs.aws.amazon.com/AmazonS3/latest/dev/RequesterPaysBuckets.html)

**Comment by jflasher** [comment]
Hi @arfon, I left a few comments. Just ping us on here again when you want this to be merged in and we'll add it.

**Comment by arfon** [comment]
Thanks for the feedback @jflasher - I think I've incorporated all your suggestions here.

**Comment by arfon** [comment]
🎉

**Pull request merged**